fix: set defaults in screenshot

jrandolf-google opened the pull request:
Also removes the override in BiDi in favor of validation in the internal `_screenshot` fn.

Fixed: #11020

OrKoN reviewed (commented):
changed to `fix:` and noticed that it does not compile yet

jrandolf committed:
chore: set defaults in screenshot

Also removes the override in BiDi in favor of validation in the internal `_screenshot` fn.
